packages/components/src/components/date-picker/date-picker-native.jsx

```diff
@@ -51,7 +51,7 @@ const Native = ({
                 {value && (
                     <Text
                         size='xs'
-                        styles={{ color: 'inherit' }}
+                        styles={{ color: 'var(--text-prominent)' }}
```

Comment:
```suggestion
                        color= 'prominent'
```

Reply:
These both solutions will create another issue, the text won't be grey out when the datepicker disabled. I replaced `prominent` with inherit to fix this in here: https://github.com/binary-com/deriv-app/pull/2840/files. Check if this issue appears again or not after reverting to `prominent`.